app/decorators/ephemera_template_decorator.rb

```diff
@@ -2,12 +2,6 @@
 class EphemeraTemplateDecorator < Valkyrie::ResourceDecorator
   self.display_attributes = []
 
-  delegate :query_service, to: :metadata_adapter
-
-  def metadata_adapter
```

Comment:
All of this moving `metadata_adapter` to the default decorator is really great, btw.